docs/Halogen/Component.md

```diff
@@ -273,4 +273,13 @@ install' :: forall s s' f f' g p p'. (Plus g, Ord p) => ParentComponentP s s' f
 A version of [`install`](#install) for use with parent components that
 `peek` on their children.
 
+#### `interpret`
+
+``` purescript
+interpret :: forall s f g g' p. (Functor g') => Natural g g' -> Component s f g p -> Component s f g' p
```

Comment:
I don't think we need to do that, as `g'` could be a `Free SomethingElse` in this formulation anyway - I just did a quick experiment on the example, added an extra layer where it `interpret`s the DSL as a `Free`-ified version before re-`interpret`ing as `Aff`.